src/main/java/com/github/alina/sesion4_challenge/ChallengeEighteen.java

```diff
@@ -0,0 +1,24 @@
+package com.github.alina.sesion4_challenge;
```

Comment:
Please try to follow for this project, recommended structure, like the one from below:

![image](https://github.com/alinaitschool/itschool/assets/120710343/681dbfe7-9a7b-4630-afd9-11faa2613db3)


Each session should have a single (master) package, and inside that package, we should have only 2 other ones: 

- practice - what we do on the course
- challenge - individual work

This structure will improve readability and will ensure we have a good composition for all the sessions that we will have.

Reply:
Fixed, thanks for valuable advices.